src/server/svr_func.c

```diff
@@ -3958,46 +3944,52 @@ set_entity_resc_sum_max(job *pjob, pbs_queue *pque, attribute *altered_resc,
  * @param[in]	pque	-	pque will point to queue structure, i.e. not be null
  * @param[in]	altered_resc	-	altered resources.
  * @param[in]	op	-	operator example- INCR, DECR
+ * @param[in]	op_flag	-	operation flag for selecting combinations of set_entity_*_sum_*()
+ * 				use ETLIM_ACC_* flag macros defined in pbs_entlim.h, ex: ETLIM_ACC_ALL
  *
  * @return	int
  * @retval	zero	: all went ok
  * @retval	PBS_Enumber	: if error, typically a system or internal error
  */
 int
 account_entity_limit_usages(job *pjob, pbs_queue *pque, attribute *altered_resc,
-	enum batch_op op)
+	enum batch_op op, int op_flag)
 {
 	int rc,ret_error = PBSE_NONE;
 
 	/* not doing NULL checks of parameters as this function is currently invoked from sane locations */
 
-	if ((rc = set_entity_ct_sum_max(pjob, pque, op)) != 0) {
-		ret_error = rc;
-		snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_ct_sum_max %s on %s %s failed with %d",
-				(op == INCR) ? "INCR" : "DECR", pque?"queue":"server", pque?pque->qu_qs.qu_name:server_name, rc);
-		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
-	}
+	if ((op_flag & ETLIM_ACC_CT_MAX) == ETLIM_ACC_CT_MAX)
```

Comment:
invoke `set_entity_ct_sum_max()` only if `ETLIM_ACC_CT_MAX`  is set i.e  both `(ETLIM_ACC_CT | ETLIM_ACC_MAX)`